builtin/plugins/user_deployer_whitelist/user_deployer_whitelist.go

```diff
@@ -326,9 +329,11 @@ func (uw *UserDeployerWhitelist) SetTierInfo(ctx contract.Context, req *SetTierI
 		return ErrNotAuthorized
 	}
 	tier := &Tier{
-		TierID: req.TierID,
-		Fee:    req.Fee,
-		Name:   req.Name,
+		TierID:     req.TierID,
```

Comment:
Validate blockrange and maxtx value. Both should be greater than 0.

Reply:
done